actionlist/app/components/action_item_body.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import {
+  StyleSheet,
+  Text,
+  View,
+  Button,
+} from 'react-native';
+
+const styles = StyleSheet.create({
+  actionContainer: {
+    backgroundColor: '#990000',
+    padding: 5,
+    borderRadius: 8,
+    width: 100,
+  },
+  routeContainer: {
+    backgroundColor: '#ffffff',
+    padding: 5,
+    borderRadius: 8,
+    borderWidth: 1,
+    borderColor: '#990000',
+    marginRight: 10,
+    width: 100,
+  },
+  actionButton: {
+    fontSize: 15,
+    color: '#ffffff',
+  },
+  routeButton: {
+    fontSize: 15,
+    color: '#990000',
+  },
+});
+
+const routeLog = () => ({});
+
+const takeAction = () => ({});
+
+const ActionItemBody = rowData => (
```

Comment:
I'm sorry, I should have said "Destructure" instead of "Spread" in my original comment, but I'm not sure what you mean when you say you have to use rowData for every key. You're only taking in one param, so you should be able to do something like 
```es6
const ActionItemBody = ({ actions, initiator, lastUpdated, processType, status }) => (
  <View>
    <Text>{processType}</Text>
    { /* ... */ }
  </View>
);
```
Does that not work?